packages/core/src/common/externals/index.ts

```diff
@@ -72,7 +73,10 @@ export interface ExternalFileSystem {
 	showFile(path: FsLocation): Promise<void>
 	stat(location: FsLocation): Promise<{ isDirectory(): boolean; isFile(): boolean }>
 	unlink(location: FsLocation): Promise<void>
-	watch(locations: FsLocation[], options: { usePolling?: boolean }): FsWatcher
+	watch(
+		locations: FsLocation[],
+		options: WatchOptions,
```

Comment:
This shouldn't directly use chokidar's `WatchOptions`, this is an interface that needs to be shared between the nodejs and browser externals